tests/experimental/nn/test_multiprocess_pipe.py

```diff
@@ -31,7 +31,7 @@
     DEVICES = [CPU_DEVICES]
 
 
-pytestmark = pytest.mark.skipif(torch_version() < (1, 9, 0), reason="requires torch version >= 1.9.0")
+pytestmark = pytest.mark.skipif(torch_version() < (1, 10, 0), reason="requires torch version >= 1.10.0")
```

Comment:
because with 1.9.0 it will fail with "can't pickle _thread.lock objects", but they used to work with 1.9.0.dev20210415+cu101

Reply:
adding the error to the comment will be really good for future references.